frontend/src/stores.ts

```diff
@@ -21,3 +25,18 @@ export const usernameStore: Readable<string | undefined> = derived(
 	}
 )
 export const superadmin = writable<String | false | undefined>(undefined)
+
+if (browser) {
+	workspaceStore.subscribe((workspace) => {
+		if (workspace) {
+			localStorage.setItem('workspace', String(workspace))
+		}
+	})
+}
+
+export function clearStores(): void {
```

Comment:
I don't see this being used for logout methods. Does logout work as expected ?

Reply:
You are right. The old implementation cleared the superadmin store when logging out, and as the login page tried to fetch unwanted data, the unhandled promise would get caught and would clear the other stores.

I'm calling `clearStores` explicitely when logging out.